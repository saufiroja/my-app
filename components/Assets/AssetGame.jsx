import React from 'react';
import Image from 'next/image';

function AssetGame() {
  return (
    <div className="absolute xl:left-[70%] xl:top-[34%] xl:block lg:left-[70%] lg:top-[34%] lg:flex md:left-[70%] md:top-[40%] md:flex sm:hidden">
      <Image src="/images/orang.svg" alt="Asset-Game" width={458} height={308} />
    </div>
  );
}

export default AssetGame;
