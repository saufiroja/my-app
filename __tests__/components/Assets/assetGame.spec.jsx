import React from 'react'
import { render, screen } from '@testing-library/react'
import AssetGame from '../../../components/Assets/AssetGame'
import renderer from "react-test-renderer"

describe('AssetGame', () => {
    it('renders a image', () => {
    render(<AssetGame />)

    const image = screen.getByRole('img', {
        name: "Asset-Game",
        })

    const incrediblesPosterImg = screen.getByAltText('Asset-Game')

        expect(image).toBeInTheDocument()
        expect(incrediblesPosterImg).toBeInTheDocument()
    })

    it('matches the snapshot', () => {
    const tree = renderer.create(<AssetGame />).toJSON()

    expect(tree).toMatchSnapshot()
    })
})