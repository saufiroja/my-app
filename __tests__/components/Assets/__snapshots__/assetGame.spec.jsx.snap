// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssetGame matches the snapshot 1`] = `
<div
  className="absolute xl:left-[70%] xl:top-[34%] xl:block lg:left-[70%] lg:top-[34%] lg:flex md:left-[70%] md:top-[40%] md:flex sm:hidden"
>
  <span
    style={
      Object {
        "background": "none",
        "border": 0,
        "boxSizing": "border-box",
        "display": "inline-block",
        "height": "initial",
        "margin": 0,
        "maxWidth": "100%",
        "opacity": 1,
        "overflow": "hidden",
        "padding": 0,
        "position": "relative",
        "width": "initial",
      }
    }
  >
    <span
      style={
        Object {
          "background": "none",
          "border": 0,
          "boxSizing": "border-box",
          "display": "block",
          "height": "initial",
          "margin": 0,
          "maxWidth": "100%",
          "opacity": 1,
          "padding": 0,
          "width": "initial",
        }
      }
    >
      <img
        alt=""
        aria-hidden={true}
        src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27458%27%20height=%27308%27/%3e"
        style={
          Object {
            "background": "none",
            "border": 0,
            "display": "block",
            "height": "initial",
            "margin": 0,
            "maxWidth": "100%",
            "opacity": 1,
            "padding": 0,
            "width": "initial",
          }
        }
      />
    </span>
    <img
      alt="Asset-Game"
      data-nimg="intrinsic"
      decoding="async"
      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
      style={
        Object {
          "border": "none",
          "bottom": 0,
          "boxSizing": "border-box",
          "display": "block",
          "height": 0,
          "left": 0,
          "margin": "auto",
          "maxHeight": "100%",
          "maxWidth": "100%",
          "minHeight": "100%",
          "minWidth": "100%",
          "objectFit": undefined,
          "objectPosition": undefined,
          "padding": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": 0,
        }
      }
    />
    <noscript>
      <img
        alt="Asset-Game"
        data-nimg="intrinsic"
        decoding="async"
        loading="lazy"
        src="/images/orang.svg"
        srcSet="/images/orang.svg 1x, /images/orang.svg 2x"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
    </noscript>
  </span>
</div>
`;
